module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -509,6 +520,24 @@ public function init()
         $this->sessionCache = $factory($namespace);
     }
 
+    /**
+     * Establish INN-Reach database connection
+     *
+     * @return PgSql\Connection|void
```

Comment:
Should this be `@return resource` for consistency with the property definition above?

Also, I don't think disabling the InnReach configuration when an exception is thrown is necessarily going to help, because there's code that's assuming `$this->innReachDb` is valid after calling this method without checking anything else. I think it might be better for the methods that use this to actually capture and check the return value, so they can act appropriately if a connection is not available for whatever reason.

Reply:
I wondered about that, too. I'll get on it.